src/base/static/components/molecules/map-legend-group.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import classNames from "classnames";
+
+import { HorizontalRule } from "../atoms/misc";
+import { Header6, Paragraph } from "../atoms/typography";
+
+import MapLegendItem from "./map-legend-item";
+
+import "./map-legend-group.scss";
+
+const MapLegendGroup = props => (
+  <div className={classNames(props.classes, "map-legend-group")}>
+    <HorizontalRule classes="map-legend-group__hr" />
+    {props.title && (
+      <Header6 classes="map-legend-group__title">{props.title}</Header6>
+    )}
+    {props.description && (
+      <Paragraph classes="map-legend-group__description">
+        {props.description}
+      </Paragraph>
+    )}
+    {props.content.map((item, i) => <MapLegendItem key={i} {...item} />)}
```

Comment:
Same here as my other comment - let's avoid the spread operator when assigning props. It will be more clear to explicitly assign the `icon`, `label`, and `swatch` props here.